docs/examples/apt-cacher-ng.md

```diff
@@ -10,10 +10,9 @@ parent = "smn_applied"
 
 # Dockerizing an apt-cacher-ng service
 
-> **Note**: 
-> - **If you don't like sudo** then see [*Giving non-root
->   access*](/installation/binaries/#giving-non-root-access).
-> - **If you're using OS X or docker via TCP** then you shouldn't use
+> **Note**: **If you don't like sudo** then see [*Giving non-root
+> access*](/installation/binaries/#giving-non-root-access)
+> **Note**: **If you're using OS X or Docker via TCP** then you shouldn't use
```

Comment:
I think we can remove the second "Note:", Starting with a new sentence (I.e., add a full stop (`.`)) should be enough